uefi-test-runner/build.py

```diff
@@ -101,27 +102,24 @@ def run_qemu(headless):
 
         # Mount the built examples directory.
         '-drive', f'format=raw,file=fat:rw:{examples_dir}',
+
+        # Connect the serial port to the host. OVMF is kind enough to connect
+        # the UEFI stdout and stdin to that port too.
+        '-serial', 'stdio',
```

Comment:
I think it's generally a good idea to have the logs of the UEFI application (whatever's printed on the virtual screen is cleared way too fast for anyone to read it), so I made this the default.